README.md

```diff
@@ -76,9 +76,9 @@ Exit
 #### Supported PowerShell Versions and Platforms
 
 - Windows PowerShell 3.0 or greater
-- PowerShell Core 6.0.2 or greater on Windows/Linux/macOS
+- PowerShell Core 6.1.0 or greater on Windows/Linux/macOS
 - Docker (tested only using Docker CE on Windows 10 1803
-  - PowerShell 6 Windows Image tags using  from [microsoft/powershell](https://hub.docker.com/r/microsoft/powershell/): `nanoserver`, `6.0.2-nanoserver`, `6.0.2-nanoserver-1709`, `windowsservercore` and `6.0.2-windowsservercore`. Example (1 warning gets produced by `Save-Module` but can be ignored):
+  - PowerShell 6 Windows Image tags using  from [microsoft/powershell](https://hub.docker.com/r/microsoft/powershell/): `nanoserver`, `6.1.0-nanoserver`, `6.1.0-nanoserver-1709`, `windowsservercore` and `6.1.0-windowsservercore`. Example (1 warning gets produced by `Save-Module` but can be ignored):
```

Comment:
Technically we would have to re-run tests for that. As part of that I might also try using a newer version of NanoServer (1809).

Reply:
I assumed changing PowerShell version would enforce changing nano server. If it isn't so I can remove this change.

Reply:
No, I meant changing `6.1.0-nanoserver-1709` to `6.1.0-nanoserver-1809`. It will probably just work but it might be worthwhile checking before we merge it

Reply:
Hmm, experiments show that the `microsoft/powershell:nanoserver` as given in the example is not working any more as the pwsh version is too old. A newer tag needs to be used, I'll have a look to tomorrow what the best tags are.

Reply:
I have updated the examples to point to the Microsoft container registry where new images are published now and updated tags, we are ready now